src/ServiceConnections/Util.h

```diff
@@ -0,0 +1,23 @@
+/**
```

Comment:
Given this whole thing is a workaround, I think it makes more sense to keep this inline in both ServiceConnections with a `// TODO: Remove once yhirose/cpp-httplib#873 is resolved`

Reply:
A comment is a great call, I'm fine inlining it given it's only two places, idk wish there was a nicer way to share the code since it's identical though.